### PR TITLE
Add discogapic and compute configs

### PR DESCRIPTION
**Pull request by andreamlin**
- `java_discogapic.yaml` is the discogapic version of `java_gapic.yaml` and points to the main class in toolkit responsible for running the discogapic pipeline.
- `artman_compute.yaml` defines the java gapic and config artifacts that are created from Discovery docs.

### CODE REVIEW

**Comment by garrettjonesgoogle** [comment]
@vam-google and @lukesneeringer could you both review this?

File: google/compute/artman_compute.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    location: git@github.com:googleapis/api-client-staging.git
+    directory_mappings:
+    - dest: generated/java/gapic-google-cloud-compute-v1
+  - name: java
+    type: GITHUB
+    location: git@github.com:GoogleCloudPlatform/google-cloud-java.git
+    directory_mappings:
+    - dest: google-cloud-compute
+- name: discogapic_config
+  type: DISCOGAPIC_CONFIG
```
Comment (andreamlin) [inline google/compute/artman_compute.yaml]: This is a new type in https://github.com/googleapis/artman/pull/409/files#diff-371ed57b95bd30de3b3331f9cc2ff446R344

File: google/compute/artman_compute.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+common:
+  api_name: compute
+  api_version: v1
+  organization_name: google-cloud
+  gapic_yaml: v1/compute_gapic.yaml
+  # TODO(andrealin): Make discovery_doc a relative path to google/compute
+  discovery_doc: google/compute/v1/compute.v1.json
+artifacts:
+- name: java_discogapic
+  type: DISCOGAPIC
```
Comment (andreamlin) [inline google/compute/artman_compute.yaml]: This is a new type in https://github.com/googleapis/artman/pull/409/files#diff-371ed57b95bd30de3b3331f9cc2ff446R335

File: gapic/lang/common.yaml [hunk 3]
```diff
@@ -7,6 +7,8 @@ common:
 java:
   gapic_language_yaml:
     - ${GOOGLEAPIS}/gapic/lang/java_gapic.yaml
+  discogapic_language_yaml:
```
Comment (andreamlin) [inline gapic/lang/common.yaml]: This is exposed as an artman parameter in https://github.com/googleapis/artman/pull/409/files#diff-d814cbce6b4cdd802d9a46d3a724fb1cR146

File: google/compute/artman_compute.yaml [hunk 4]
```diff
@@ -0,0 +1,24 @@
+common:
+  api_name: compute
+  api_version: v1
+  organization_name: google-cloud
+  gapic_yaml: v1/compute_gapic.yaml
+  # TODO(andrealin): Make discovery_doc a relative path to google/compute
+  discovery_doc: google/compute/v1/compute.v1.json
```
Comment (andreamlin) [inline google/compute/artman_compute.yaml]: Ideally this line would be 
`discovery_doc: google/compute/v1/compute.v1.json`
but the path resolution hasn't been enabled in artman

**Comment by andreamlin** [comment]
Changed `java_discogapic` back to `java_gapic`; changed `discogapic_config` back to `gapic_config`

**Comment by lukesneeringer** [comment]
Note that copybara might end up plowing over this.

**Comment by garrettjonesgoogle** [comment]
Pursuant to @lukesneeringer 's comment, this probably needs to be submitted internally actually.

**Comment by andreamlin** [comment]
Checking these files into discovery-artifact-manager instead: https://github.com/googleapis/discovery-artifact-manager/pull/77.

**Pull request closed without merging**